scm/helper.go

```diff
@@ -51,6 +55,8 @@ var (
 	ErrNotMember = errors.New("user is not a member of the organization")
 	// ErrNotOwner indicates that user has no admin rights in the requested organization.
 	ErrNotOwner = errors.New("user is not an owner of the organization")
+	// ErrMissingInstallation indicates that GitHub application is not installed on organization.
+	ErrMissingInstallation = connect.NewError(connect.CodePermissionDenied, errors.New("github application is not installed on the course organization"))
```

Comment:
I'm not a fan of making the `scm` package (or any other packages for that matter) depend on the connect package. I would prefer to keep the connect dependencies in the `web` package and its sub-packages.

Reply:
I'm not sure what the frontend will see if we just return a regular error. But the message itself should be fine. Whether or not we need an error code such as permission denied, I don't know.

Reply:
If an error has no code, connect will attach the default `CodeUnknown` to it. Is it ok?

Reply:
Or, because there is only one error that the `GetSCM*` methods can return, we can add the code with `connect.NewError(connect.CodeNotFound, err)` in `quickfeed_service.go` before passing it to the frontend.

Reply:
Yes, absolutely.